mopidy_mpd/session.py

```diff
@@ -29,6 +29,14 @@ def on_start(self):
     def on_line_received(self, line):
         logger.debug("Request from %s: %s", self.connection, line)
 
+        # To prevent CSRF attacks, lines with an invalid command syntax are immediately dropped.
+        command = line.split(" ")[0]
```

Comment:
Why do we need to split?

Reply:
I only check against the actual command name, as the possible characters for that are known. So I split the line at the spaces and then grab the first element (the command name) to check the validity.

But I think the protocol actually supports any whitespace between commands and arguments so I should probably change this.

Reply:
The first character in the line is also the first character of the command.

Reply:
@kingosticks I chose validate the complete command name here. But I could return to just testing the first character like `mpd` if that's preferred.

Reply:
Yes, I think that's better rather than trying to do more complicated parsing here.